docs/site/Model-relations.md

```diff
@@ -0,0 +1,27 @@
+- Relations are defined between models via relational properties on the source
+  {model, repository}. The relation metadata from the relational property should be used to construct the constrained repository instance (we need the source instance, target repository and constraint to do so).
+- The relational access needs to be configured/resolved between repositories.
+  For example, a CustomerRepository and an OrderRepository are needed to perform
+  CRUD operations for a given customer and his/her orders.
+- Repository interfaces for relations define the available CRUD methods based on
+  the relation type. Default relation repository classes implement those
+  interfaces and enforce constraints on them using utility functions.
+- We should support LB3 inclusion for related models which returns an array of
+  related model instances as plain data objects. The array is populated only
+  when the caller specifies the `includes` inside the filter. This would be a
+  plural of the related model, in the case of customer has many orders, it would
```

Comment:
This should be the value provided in `as` property of the releation. By default, juggler is setting `as` to the plural form of the target model.